generators/app/templates/webapp/Component.ts

```diff
@@ -1,19 +1,18 @@
 import UIComponent from "sap/ui/core/UIComponent";
-import { support } from "sap/ui/Device";
+import Device from "sap/ui/Device";
```

Comment:
This change has to be done depending on the UI5 version. For the tutorial it's fine like this, but in general people will choose to develop against older stable versions of UI5.

Reply:
Good catch